buildSrc/build.gradle.kts

```diff
@@ -0,0 +1,19 @@
+plugins {
+    `kotlin-dsl`
+    `java-library`
+}
+
+repositories {
+    mavenLocal()
+    mavenCentral()
+    gradlePluginPortal()
+    jcenter()
+    maven(url = "https://files.minecraftforge.net/maven")
```

Comment:
This will also need the Sponge Maven repository once merged, although I don't know where in the order it ought to go (if it's above `mavenLocal()`, published versions would take priority; if it's below, unpublished versions would take priority). Same thing with SAPI, SC, and SV.